src/snowflake/cli/_plugins/nativeapp/entities/application.py

```diff
@@ -769,6 +772,7 @@ def stream_events(
         except KeyboardInterrupt:
             return
 
+    @span("get_snowsight_url_for_app")
```

Comment:
IMO this doesn't need to be recorded

Reply:
If you take a look at the example payload in the PR description it's actually a sizeable chunk, around 3s~ which is bigger than some of the other spans, and when it's not there it seems like the trace is missing something since the total execution time isn't roughly equal to the sum of the spans' execution times. I'm not against removing it if we feel it's unnecessary and we agree we can infer what's happening in the blind spot, just wanted to bring attention to it and see if you have any other thoughts